niworkflows/anat/ants.py

```diff
@@ -287,6 +287,8 @@ def init_brain_extraction_wf(name='brain_extraction_wf',
             n_iterations=[50] * 5, convergence_threshold=1e-7, shrink_factor=4,
             bspline_fitting_distance=bspline_fitting_distance),
         n_procs=omp_nthreads, name='inu_n4_final', iterfield=['input_image'])
+    if _ants_version and parseversion(_ants_version) >= Version('2.1.0'):
+        inu_n4_final.inputs.rescale_intensities = True
```

Comment:
Does this indicate that we're diverging from `antsBrainExtraction.sh`, or is this a change they also make in 2.1.0? Would we consider the result just as valid with old ANTs, and if not, should we just set this and let the `min_ver` check fail?

Reply:
This is applied to a refinement step not found in the original script. Looking at it with perspective, we should've taken out this final bias correction away from this workflow.

However, to keep minimal changes in fMRIPrep, we stuffed this in the workflow because it seemed better not to modify the API.

In this context, adding the rescale flag makes sense for this final INU step (please note I didn't include it in the initial INU-correction which comes from the original ANTs' script).

Reply:
Okay, but shouldn't we set it to`True` always, and error if bad version?

Reply:
Not having the flag set would not change the output dramatically. I would recommend using it, but would not prevent people from running the workflow with older versions of ANTs. Since this is a "general purpose" workflow, I'd keep the option for the low-risk it entails.

Reply:
I'm adding a warning - would that address your concern?

Reply:
Sure.